boards/BMS/Inc/Io/Io_LTC6813/Io_SharedLTC6813.h

```diff
@@ -68,22 +68,22 @@ void Io_NewLTC6813_InitSpiHandle(SPI_HandleTypeDef *spi_handle);
  * PEC15 bytes from
  * @return The PEC15 code generated from the contents of data_buffer
  */
-uint16_t Io_NewLTC6813_CalculateRegGroupPec15(uint8_t *data_buffer);
+uint16_t Io_SharedLTC6813_CalculateRegGroupPec15(uint8_t *data_buffer);
```

Comment:
should make pointer const since we're not modifying the data buffer

Reply:
im assuming you meant pointer to a `const uint8_t` here, added